rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1094,6 +1083,44 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		}
 	};
 
+	auto on_lib_button_clicked = [this, on_search_box_text_changed](int id)
+	{
+		on_search_box_text_changed({}); // Disable search filter
+
+		std::vector<QListWidgetItem*> items;
+
+		// duplicate current items, we need clones to preserve checkstates
+		for (int i = 0; i < ui->lleList->count(); i++)
+		{
+			items.push_back(ui->lleList->item(i)->clone());
+		}
+
+		for (uint i = 0; i < items.size(); i++)
```

Comment:
I'd simply integrate this into the other lambda and just get the text and button Id in the lambda itself or as arg.

Reply:
Fixed, fixed.